pycheribuild/projects/cross/opensbi.py

```diff
@@ -140,7 +140,11 @@ def install(self, **kwargs):
         if self.crosscompile_target.is_hybrid_or_purecap_cheri() and not self.build_dir_suffix:
             # Install into the QEMU firware directory so that `-bios default` works
             qemu_fw_dir = self._qemu_install_dir() / "share/qemu/"
-            suffix = "cheristd" if self.crosscompile_target.is_cheri_purecap() else "cheri"
+            suffix = ""
+            if self.crosscompile_target.is_cheri_purecap():
+                suffix = "cheri"
+                if self.config.riscv_cheri_isa == RiscvCheriISA.STD:
+                    suffix += "std"
```

Comment:
Isn't anything in "CHERI-Alliance SDK" supposed to be standard CHERI-RISC-V by default? Do we expect to build other non-std CHERI-RISC-V with this SDK at all?